content/2024/chinese/program-committee.zh.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Committee Members"
+seo_title: "Jakarta One Livestream - Chinese 2022 | Jakarta EE Software | Cloud Native"
```

Comment:
The `seo_title` should be corrected to 2024

Reply:
Done.